src/bp/ui-studio/src/web/views/OneFlow/diagram/VariableTypesForm/EnumForm.tsx

```diff
@@ -23,6 +25,7 @@ const EnumForm: FC<Props> = ({
   customKey,
   contentLang,
   formData,
+  allEntities,
```

Comment:
Why not just "entities" ?

Reply:
because it's all the entities of all the enumerations and not just the current editable one.

I don't mind to change for entities tho.